lib/active_record/connection_adapters/mssql_adapter.rb

```diff
@@ -1 +0,0 @@
-require 'arjdbc/mssql'
```

Comment:
do we not want the `adapter: mssql` magic to work? (and require sqlserver adapter parts)